instructors.md

```diff
@@ -7,196 +7,245 @@ subtitle: Instructor's Guide
-    commit it,
-    and push the change to the remote repository
-    so that the owner can then retreive it,
-    and vice-versa.
-    The role playing between the instructors can get a bit "dramatic" in the conflicts part of the lesson
-    if the instructors want to inject some humor into the room.
-
-*   If you don't have two projectors,
-    have two instructors at the front of the room.
-    Each instructor does their piece of the collaboration demonstration on their own computer
-    and then passes the projector cord back and forth with the other instructor
-    when it's time for them to do the other part of the collaborative workflow.
-    It takes less than 10 seconds for each switchover,
-    so it doesn't interrupt the flow of the lesson.
-
-    And of course it helps to give each of the instructors a different-colored hat,
-    or put different-colored sticky notes on their foreheads.
+*   Decide in advance whether all the learners will work in one shared
+    repository, or whether they will work in pairs (or other small groups) in
+    separate repositories.  The former is easier to set up; the latter runs
+    more smoothly.
+
+*   Role playing between two instructors can be effective when teaching the
+    collaboration and conflict sections of the lesson.  One instructor can play
+    the role of the repository owner, while the second instructor can play the
+    role of the collaborator.  If it is possible, try to use two projectors so
+    that the computer screens of both instructors can be seen.  This makes for
+    a very clear illustration to the students as to who does what.
+
+*   It is also effective to pair up students during this lesson and assign one
+    member of the pair to take the role of the owner and the other the role of
+    the collaborator.  In this setup, challenges can include asking the
+    collaborator to make a change, commit it, and push the change to the remote
+    repository so that the owner can then retrieve it, and vice-versa.  The
+    role playing between the instructors can get a bit "dramatic" in the
+    conflicts part of the lesson if the instructors want to inject some humor
+    into the room.
+
+*   If you don't have two projectors, have two instructors at the front of the
+    room.  Each instructor does their piece of the collaboration demonstration
+    on their own computer and then passes the projector cord back and forth
+    with the other instructor when it's time for them to do the other part of
+    the collaborative workflow.  It takes less than 10 seconds for each
+    switchover, so it doesn't interrupt the flow of the lesson.
+    And of course it helps to give each of the instructors a different-colored
+    hat, or put different-colored sticky notes on their foreheads.
+
+*   If you're the only instructor, the best way to create is clone the two
+    repos in your Desktop, but under different names, e.g., pretend one is your
+    computer at work: 
+
```

Comment:
Suggest maybe using two computers, even if only one instructor? Especially if two projectors are available, this might make it considerably more obvious what is being done from each version of the repo.